test-utils/runtime-tester/src/run_test.rs

```diff
@@ -51,15 +52,17 @@ impl Scenario {
 
             let start_time = Instant::now();
 
-            last_block = env.clients[0].produce_block(block.height).unwrap().unwrap();
+            last_block = env.clients[0]
+                .produce_block(block.height)?
+                .ok_or(Error::Other(String::from("No block has been produced")))?;
```

Comment:
Nit
```suggestion
                .ok_or_else(|| Error::Other(String::from("No block has been produced")))?;
```

The original allocates a string every time, the new version only on the sad path.